ena-submission/Snakefile

```diff
@@ -18,20 +18,22 @@ with open("results/config.yaml", "w") as f:
     f.write(yaml.dump(config))
 
 LOG_LEVEL = config.get("log_level", "INFO")
-ORGANISMS = config['organisms'].keys()
+ORGANISMS = config["organisms"].keys()
 
 
 rule submit_all_external_metadata:
     input:
-        expand("results/submitted_{organism}.json", organism=ORGANISMS)
-    
+        expand("results/submitted_{organism}.json", organism=ORGANISMS),
+
+
 rule submit_external_metadata:
```

Comment:
This will be added in a later PR - for the moment you can test it out using data in the test folder

Reply:
I realized that after I left the comment, thanks!